src/main/java/sparksoniq/io/json/JiqsItemParser.java

```diff
@@ -36,7 +36,7 @@
     public static Item getItemFromObject(Object object, IteratorMetadata metadata) {
         if (object != null) {
             if (object instanceof String)
-                return new StringItem(object.toString());
+                return new StringItem(object.toString().intern());
```

Comment:
StackOverflow wisdom: "be aware that the cache is maintained by JVM in permanent memory pool which is usually limited in size compared to heap so you should not use intern if you don't have too many duplicate values."

I wonder how large this can grow in real life...

Reply:
Do you have a link to this StackOverflow page, in particular, how old is it? I had this in mind, but then read that newer versions of Java (I think starting with 7) store these Strings in the JVM, no longer in the permanent memory pool.